float64[6] e
float64[6] d_e
float64[12] yita
float64[12] yita_hat
float64[12] yita_tilde
float64[6] y_alpha
float64[6] y_beita
float64[6] d_alpha_hat
float64[6] alpha_t
float64[6] d_beita_hat
float64[6] beita_t
float64[6] theta_hat
float64[6] u_1
float64[6] b_d

float64[3] vCmdGlobal
float64[3] velbody
float64 pcd_2
float64 posBody_2
float64[3] wCmdGlobal
float64[3] lowState_w

